src/lib/request.js

```diff
@@ -70,7 +70,7 @@ export default function (relativeUrl, opts, callback) {
     }
 
     let url = relativeUrl;
-    if (options.requestOptions.baseUrl) {
+    if (options.requestOptions.baseUrl.length > 0) {
```

Comment:
Considering the default value for `baseUrl` is `null`, this change could introduce a regression because `length` is not a method in `null`.